CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changes
 
+## 0.36.0 (2025-01-24)
+
+Feature release
+
+- [`OF#5006`][OF#5006] It's now possible to fill in manually the street name and the city in the
+  `addressNL` component in case these are not available from the external API call.
```

Comment:
```suggestion
- [`OF#5006`][OF#5006] Updated stories for manual AddressNL city/street name fallback in case autofill isn't available.
```
I don't think anything effectively changed in this component? There are no new configuration options for form designers etc.